### ISSUE TITLE
Syntax error with JDK10

### ISSUE THREAD

**Issue by larsrh** [paulp/sbt-extras#212]
Environment: Arch Linux, java-10-openjdk, sbt-extras daa79533f12a086deba7e717b7b43a99e601d4b6

When running `sbt` without arguments, I get this:

```
$ sbt
/home/lars/bin/sbt: line 205: [[: .: syntax error: operand expected (error token is ".")
```

But `sbt` proceeds to start as normal.

**Comment by paulp** [comment]
#213

**Comment by larsrh** [comment]
Fixed by #213.
